docs/source/tensorbay_cli/tbrn.rst

```diff
@@ -1,3 +1,96 @@
 #########################
  TensorBay Resource Name
 #########################
+
+TensorBay Resource Name(TBRN) uniquely defines the resource stored in TensorBay.
+TBRN begins with ``tb:``. It can represent different levels of resources in TensorBay:
```

Comment:
the word `level` is a new word in our doc, can we use another word or just remove it?

Reply:
Delete the last sentence.